docs/notifications.md

```diff
@@ -0,0 +1,35 @@
+# Notification Development Instructions
+
+The purpose of this document is to provide ZAP developers with simple instructions on how to add a notification.
+
+## ADDING a Notification:
+
+In order to add a notification, call the setNotification function in zap/src-electron/query-notification.js. setNotification takes the following arguments:
+
+- db => database object (object)
+- type => the type of notification you wish to define (string)
+- status => the message of the notification (string)
+- sessionId => the session Id (string)
+
+For example,
+
+```
+async function iscDataLoader(db, state, sessionId) {
+  let endpointTypes = state.endpointTypes
+  let promises = []
+  await notification.setNotification(
+    db,
+    'UPGRADE',
+    'ISC FILE UPGRADED TO ZAP FILE. PLEASE SAVE AS TO SAVE OFF NEWLY CREATED ZAP FILE.',
+    sessionId,
+    1
+  )
+```
+
+- The front end code does not need to be changed when adding a notification.
```

Comment:
You can get rid of this comment on the front end.

Reply:
okay